packages/panzer/disc-fe/src/lof/Panzer_BlockedEpetraLinearObjFactory.hpp

```diff
@@ -241,6 +241,7 @@ class BlockedEpetraLinearObjFactory : public LinearObjFactory<Traits>
 
    //! Get the domain vector space (x and dxdt)
    Teuchos::RCP<const Thyra::VectorSpaceBase<double> > getGhostedThyraDomainSpace() const;
+   Teuchos::RCP<const Thyra::VectorSpaceBase<double> > getGhostedThyraDomainSpace2() const;
```

Comment:
For the time being, there is now a version 2 of various functions, which will be used in conjunction with `EpetraVector_ReadOnly_GlobalEvaluationData`s.  If `EpetraLinearObjContainer`s are in use instead, the original versions of these routines will be used.  The goal would be to eventually rename all the version 2 routines to the original names.